credentialz/credentialz.proto

```diff
@@ -224,6 +224,16 @@ service Credentialz {
   // Get the current public keys from the host. Each configured key will be
   // returned in the provided list.
   rpc GetPublicKeys(GetPublicKeysRequest) returns (GetPublicKeysResponse);
+
+  // Add a system role account.
```

Comment:
Would be good to add a strict definition what a "system role account" is.

OpenConfig has `SYSTEM_ROLE_ADMIN` https://github.com/openconfig/public/blob/db78d44e93a97cef4db27f82bba3fba454e907a5/release/models/system/openconfig-aaa-types.yang#L77-L84, but I have a feeling that you mean something else in this context.

Reply:
Is it the case, avanpo, that you mean here:
 "Add an account to the NOS, but not to the underlying unix system."
 
 This maybe an operator account:
 
    operator - robot/user responsible for interface up/down/troubleshooting

or an administrative account:

   god - the user with all power/privilege on the device